Jenkinsfile

```diff
@@ -388,9 +391,6 @@ stage('Default build') {
 					"""
 				}
 			}
-			catch (CredentialNotFoundException e) {
-				echo "No SonarCloud token configured - skipping SonarCloud report. Error was: ${e}"
-			}
```

Comment:
See, you end up removing this catch ;).

Reply:
Yeah, if someone explicitly configured a sonar organization and credentials are not found, then it's definitely an error :)